cmake/modules/CMakeLists.txt

```diff
@@ -87,10 +88,6 @@ install(
   EXPORT AIETargets
   DESTINATION ${AIE_INSTALL_PACKAGE_DIR}
   COMPONENT aie-cmake-exports)
-install(
```

Comment:
Transcribing offline discussion: this is necessary to support downstream users because of how `add_mlir_library` works.